lib/cchain/provider/provider.go

```diff
@@ -179,9 +183,18 @@ func (p Provider) stream(
 	srcChain := p.chainNamer(chainVer)
 	ctx := log.WithCtx(in, "src_chain", srcChain, "worker", workerName)
 
+	// cachedOffsetHeight is used to store the height at which the previous attestation offset was found
```

Comment:
nit: `// Cache the previous batch fetch height, to prevent lookback searching each time`